### PR TITLE
New code added: uploading js assignment 6

### PR DESCRIPTION
**Pull request by poonam-khode**

### CODE REVIEW

File: poonam/JS/Assignment6_js/que5.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <script>
+            var input=[];
+            function inputnum(){
+                let num = true;
+                while(num) {
+                    let n = prompt('Enter a Number:');
+                    if (n != 'x') {
+                        input.push(parseInt(n));
+                    }
+                else {
```
Comment (VIP5614) [inline poonam/JS/Assignment6_js/que5.html]: All correct but else space 4 column

**Review by VIP5614 — CHANGES_REQUESTED** [review]

**Review by VIP5614 — APPROVED** [review]

File: poonam/JS/Assignment6_js/que5.html (hunk 1 above)
Comment (poonam-khode) [reply poonam/JS/Assignment6_js/que5.html]: done

Comment (VIP5614) [reply poonam/JS/Assignment6_js/que5.html]: yes Done

**Review by sanketmod — APPROVED** [review]

**Review by pratikshaghule — APPROVED** [review]

File: poonam/JS/Assignment6_js/que1.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            input[i]=prompt("Enter name:");
+        }
+        document.writeln("List of names: "+input);
+        document.writeln("<br>Name List:");
+        for(let i=0; i<=4; i++){
+            document.writeln("<br>Name: "+input[i]);
+        }
+    </script>
+    </body>
+</html>
```
Comment (SnehaMakade) [inline poonam/JS/Assignment6_js/que1.html]: Add new end line in all files.

Comment (poonam-khode) [reply poonam/JS/Assignment6_js/que1.html]: which line mam?

**Review by Priyankashelke2411 — APPROVED** [review]

**Review by Priyankashelke2411 — APPROVED** [review]

File: poonam/JS/Assignment6_js/que1.html (hunk 2 above)
Comment (SnehaMakade) [reply poonam/JS/Assignment6_js/que1.html]: like in all files after closing of </html> add new line .

File: poonam/JS/Assignment6_js/que1.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>que1</title>
+    </head>
+    <body>
+        <h1>Name List</h1>
+        <script>
+        var input=[];
+        for(let i=0; i<=4; i++){
+            input[i]=prompt("Enter name:");
```
Comment (SnehaMakade) [inline poonam/JS/Assignment6_js/que1.html]: add proper lable for asking input like enter one by one five names in prompt box.

File: poonam/JS/Assignment6_js/que2.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>que2</title>
+        <script>
+            var i=0;
+            var na=Array();
```
Comment (SnehaMakade) [inline poonam/JS/Assignment6_js/que2.html]: variable names should be proper correct variable names in all files

File: poonam/JS/Assignment6_js/que2.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+            function add(){
+                document.getElementById("b").innerHTML="Name list by using join ~ is : ";
+                na[i]=document.getElementById("a").value;
+                i++;
+                document.getElementById("a").value="";
+                document.getElementById("b").innerHTML+=na.join("~");
+            }
+            function asc_sort(){
```
Comment (SnehaMakade) [inline poonam/JS/Assignment6_js/que2.html]: fnction name should be proper like ascending_sort, descending_sort

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
change the variable and function names. give suitable name to both according assignment requirement.

File: poonam/JS/Assignment6_js/que2.html [hunk 6]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>que2</title>
+        <script>
+            var i=0;
```
Comment (SnehaMakade) [inline poonam/JS/Assignment6_js/que2.html]: there should be space before and after any operator like " =" operator or +, - and all type operator do this change in all files and space should be there after comma, not before

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
add space before and after operator in all files

**Review by SnehaMakade — APPROVED** [review]

**Pull request merged**